kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -132,16 +175,33 @@
 
   /**
    * Create a {@link Dataset} for the given dataset or view URI.
+   * {@code create} returns an empty dataset. You can use 
+   * {@link DatasetWriter} to populate your dataset.
```

Comment:
Best practice is to use a short sentence to explain what the method does, then start a new paragraph for additional info.

I don't know that we need to link to `DatasetWriter`. What is the value?